install-script.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+PWD=$(pwd)
+
+# first, backup any existing scripts if they are not symlinks
+if [[ -d "$HOME/.vim" && ! -L "$HOME/.vim" ]]; then
+  echo -e "moving ~/.vim to ~/.vim_backup"
+  mv ~/.vim ~/.vim_backup
+else
+  echo "$HOME/.vim has already been moved, is a symlink, or does not exist"
```

Comment:
Would you prefer to `exit` in this case?

Also, to avoid a partially broken installation, what do you think about first doing all the sanity checks once at the top of the script, and if anything looks out of place - exit with the warning. Otherwise, you can do all the `mv` etc in one shot atomically.